src/main/java/com/daml/extensions/testing/junit5/SandboxTestExtension.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2020 Digital Asset (Switzerland) GmbH and/or its affiliates
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.daml.extensions.testing.junit5;
+
+import org.junit.jupiter.api.extension.AfterEachCallback;
+import org.junit.jupiter.api.extension.BeforeEachCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.platform.commons.support.AnnotationSupport;
+
+import java.io.IOException;
+import java.lang.reflect.Field;
+import java.util.List;
+import java.util.concurrent.TimeoutException;
+
+public class SandboxTestExtension implements AfterEachCallback, BeforeEachCallback {
+
+  @Override
+  public void afterEach(ExtensionContext context) {
+    context
+        .getTestInstance()
+        .ifPresent(
+            testInstance -> {
+              List<Field> sandboxFields =
+                  AnnotationSupport.findAnnotatedFields(testInstance.getClass(), TestSandbox.class);
+              for (Field sandboxField : sandboxFields) {
+                try {
+                  Sandbox sandbox = (Sandbox) sandboxField.get(testInstance);
+                  sandbox.stop();
+                } catch (IllegalAccessException e) {
+                  e.printStackTrace();
```

Comment:
Shouldn't we just rather throw some runtime (unchecked) exception?
I am not sure printing stack trace and continuing would make anyone's life easier.